cmd/draft/connect.go

```diff
@@ -96,41 +105,85 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 
 	var connectionMessage = "Your connection is still active.\n"
 
+	exportEnv := make(map[string]string)
+
 	// output all local ports first - easier to spot
 	for _, cc := range connection.ContainerConnections {
 		for _, t := range cc.Tunnels {
-			err = t.ForwardPort()
-			if err != nil {
+			if err = t.ForwardPort(); err != nil {
 				return err
 			}
-			m := fmt.Sprintf("Connect to %v:%v on localhost:%#v\n", cc.ContainerName, t.Remote, t.Local)
-
-			connectionMessage += m
-			fmt.Fprintf(cn.out, m)
-		}
-	}
-
-	for _, cc := range connection.ContainerConnections {
-		readCloser, err := connection.RequestLogStream(deployedApp.Namespace, cc.ContainerName, cn.logLines)
-		if err != nil {
-			return err
+			if export {
+				var (
+					application = strings.Replace(strings.ToUpper(deployedApp.Name), "-", "_", -1)
+					container   = strings.ToUpper(cc.ContainerName)
+					prefix      = fmt.Sprintf("%s_%s", application, container)
+				)
+				exportEnv[prefix+"_SERVICE_HOST"] = fmt.Sprintf("localhost")
+				exportEnv[prefix+"_SERVICE_PORT"] = fmt.Sprintf("%#v", t.Local)
+			} else {
+				m := fmt.Sprintf("Connect to %v:%v on localhost:%#v\n", cc.ContainerName, t.Remote, t.Local)
+				connectionMessage += m
+				fmt.Fprintf(cn.out, m)
+			}
 		}
-
-		defer readCloser.Close()
-		go writeContainerLogs(cn.out, readCloser, cc.ContainerName)
 	}
 
-	stop := make(chan os.Signal, 2)
+	stop := make(chan os.Signal, 1)
+	done := make(chan struct{})
 	signal.Notify(stop, os.Interrupt, syscall.SIGTERM)
 	go func() {
 		<-stop
-		os.Exit(0)
+		close(done)
 	}()
 
-	for {
-		time.Sleep(5 * time.Minute)
-		fmt.Fprintf(cn.out, connectionMessage)
+	if export {
+		for env, val := range exportEnv {
+			fmt.Fprintf(os.Stdout, "export %s=%s\n", env, val)
```

Comment:
Can we make this cross-platform compatible?

The good news is that we can just re-apply the same logic `gofish tank` uses here :)

https://github.com/fishworks/gofish/blob/master/cmd/gofish/tank.go
https://github.com/fishworks/gofish/blob/master/cmd/gofish/tank_unix.go
https://github.com/fishworks/gofish/blob/master/cmd/gofish/tank_windows.go